client/src/pages/SignUpModal.tsx

```diff
@@ -1,10 +1,10 @@
-import './Modal.scss';
-import { OverlayTrigger, Popover } from 'react-bootstrap';
-import React, { FunctionComponent } from 'react';
-import CommonModal from '../components/organisms/Modal';
-import { createNewAccount } from '../services/auth';
-import { Redirect } from 'react-router-dom';
-import {useState} from 'react';
+import "./Modal.scss";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import React, { FunctionComponent } from "react";
+import CommonModal from "../components/organisms/Modal";
+import { createNewAccount } from "../services/auth";
+import { Redirect } from "react-router-dom";
+import { useState } from "react";
```

Comment:
nit: the `useState` import can go in the same import as React
ex. `import React, { FunctionComponent, useState } from "react";`